src/ci/docker/host-x86_64/x86_64-gnu-llvm-9/Dockerfile

```diff
@@ -39,22 +39,13 @@ ENV SCRIPT python2.7 ../x.py --stage 2 test --exclude src/tools/tidy && \
            # the PR is approved and tested for merging.
            # It will also detect tests lacking `// EMIT_MIR_FOR_EACH_BIT_WIDTH`,
            # despite having different output on 32-bit vs 64-bit targets.
-           #
-           # HACK(eddyb) `armv5te` is used (not `i686`) to support older LLVM than LLVM 9:
-           # https://github.com/rust-lang/compiler-builtins/pull/311#issuecomment-612270089.
-           # This also requires `--pass=build` because we can't execute the tests
-           # on the `x86_64` host when they're built as `armv5te` binaries.
-           # (we're only interested in the MIR output, so this doesn't matter)
-           python2.7 ../x.py --stage 2 test src/test/mir-opt --pass=build \
-                             --host='' --target=armv5te-unknown-linux-gnueabi && \
+           python2.7 ../x.py --stage 2 test src/test/mir-opt \
+                             --host='' --target=i686-unknown-linux-gnueabi && \
```

Comment:
The target is called `i686-unknown-linux-gnu` (without the eabi, that's an ARM specific thing).

Reply:
Yes, I was a bit over-eager with the copy and paste there, it's fixed now (though with a force-push instead of a commit because I didn't see your review in time, sorry)